doc/contributor/howto-guide-declare-a-library-ga.md

```diff
@@ -36,11 +36,12 @@ For a generated library there are 3 critical checks:
 In addition, we (the Cloud C++ team) require a simple `quickstart.cc` for each
 library.  This program is typically created when the library is generated.
 
-### `BUILD.bazel`
+### `cmake/GoogleCloudCppFeatures.cmake`
 
-Update the top-level `BUILD.bazel` file. Move the library from
-`EXPERIMENTAL_LIBRARIES` to `TRANSITION_LIBRARIES`. Do this first as it helps
-automate the following steps.
+Update `cmake/GoogleCloudCppFeatures.cmake`. Move the library from
+`GOOGLE_CLOUD_CPP_EXPERIMENTAL_LIBRARIES` to
+`GOOGLE_CLOUD_CPP_TRANSITION_LIBRARIES`. Do this first as it helps automate the
+following steps.
 
 ```shell
 mapfile -t ga < <(bazel --batch query \
```

Comment:
There is an easier way to do this now:

```
mapfile -t ga < <(cmake -DCMAKE_MODULE_PATH="${PWD}/cmake" -P cmake/print-ga-libraries.cmake | 
  grep -E -v 'storage|bigtable|spanner|pubsub|common|grpc_utils')
```

Reply:
Fixed, thanks. I had to add `2>&1` to get the thing to work.